resources/sugarcrm/modulename.cpp

```diff
@@ -0,0 +1,43 @@
+#include <QString>
+#include <QMap>
+#include <KDebug>
+#include "modulename.h"
+
+struct ModuleEntry {
+    const char* name;
+    const ModuleName module;
+};
+
+static const ModuleEntry s_moduleNames[] = {
+    {"Accounts", Accounts},
+    {"Opportunities", Opportunities},
+    {"Campaigns", Campaigns},
+    {"Leads", Leads},
+    {"Contacts", Contacts},
+    {"Documents", Documents},
+    {"Emails", Emails},
+    {"Notes", Notes},
+    {"Tasks", Tasks}
+};
+
+static const int count = sizeof(s_moduleNames) / sizeof(*s_moduleNames);
+
+QString moduleNameToString(ModuleName moduleName)
```

Comment:
It's just confusing that "ModuleName" is the enum. How about renaming it to just "Module" ?